parser/proto.y

```diff
@@ -604,43 +573,31 @@ compactOptionDecls : compactOption {
 		$$ = &compactOptionList{$1, $2, $3}
 	}
 
-compactOption: optionName '=' constant {
+compactOption: optionName '=' optionValue {
 		refs, dots := $1.toNodes()
 		optName := ast.NewOptionNameNode(refs, dots)
 		$$ = ast.NewCompactOptionNode(optName, $2, $3)
 	}
 
-group : _REQUIRED _GROUP name '=' _INT_LIT '{' messageDecls '}' {
-		$$ = ast.NewGroupNode($1.ToKeyword(), $2.ToKeyword(), $3, $4, $5, nil, $6, $7, $8)
-	}
-	| _OPTIONAL _GROUP name '=' _INT_LIT '{' messageDecls '}' {
-		$$ = ast.NewGroupNode($1.ToKeyword(), $2.ToKeyword(), $3, $4, $5, nil, $6, $7, $8)
+groupDecl : fieldCardinality _GROUP identifier '=' _INT_LIT '{' messageElements '}' {
```

Comment:
Can you open a PR against github.com/bufbuild/protobuf.com to fix the spec issue here (where it incorrectly indicates that the cardinality is optional)?

Reply:
Just opened a [PR](https://github.com/bufbuild/protobuf.com/pull/24) to fix this.